config/file.go

```diff
@@ -66,7 +74,9 @@ func resolveConfigFilePath(path string) (string, error) {
 	// Otherwise, search for the config/ folder using the same heuristics as above, and build
 	// an absolute path anchored there and joining the given input path (or plain filename).
 	if configFolder, found := fileutils.FindDir("config"); found {
-		return filepath.Join(configFolder, path), nil
+		if configFile := fileutils.FindFile(filepath.Join(configFolder, path)); configFile != "" {
```

Comment:
Isn't this the same as L64? Wondering what's the difference here.

Reply:
Reverted it. It was checking if the parent config folder exits, rather than searching file itself.